### ISSUE TITLE
Exclude Dependencies

### ISSUE THREAD

**Issue by Sly777** [mzgol/check-dependencies#14]
Hi all,

Can we exclude some dependencies for checking? For example, "color-thief" plugin doesn't have semver tag and check-dependencies gives error for that. This feature will fix problematic dependencies.

Thanks!

**Comment by mzgol** [comment]
To be sure I understand correctly - you want to have the `checkGitUrls` option enabled so that it works on other Git URLs but you want it skipped for specific packages?

If that's what you want, I'd accept such a feature. I don't have a lot of time for it currently, though, so PR welcome!

**Comment by andineck** [comment]
I get the following error on my mac ;-(

```
  error: 
   [ 'Package .DS_Store installed, though it shouldn\'t be',

```

A ignore option would help.

**Comment by mzgol** [comment]
@andineck Hmm, I should really just ignore all folders starting with a dot (I already do that with `.bin`).

**Comment by andineck** [comment]
I think that would be a good idea and would definitively help in this case. 
A ignore Array option would be a more general solution to ignoring also "valid" npm packages.

**Comment by andineck** [comment]
Thank you! @mzgol

**Comment by mzgol** [comment]
I've published a beta version with the fix, install with `npm -g i check-dependencies@beta`. I'll publish a final version soon, just waiting for some feedback at #13 first.

BTW, out of curiosity - do you all still need Node.js 0.10 support or would 0.12+ be enough?

**Comment by andineck** [comment]
for me 0.12+ is enough. Although I have closed systems with 0.8 running (no more updates), I think it is more important to move forward to 4.0

**Comment by mzgol** [comment]
`0.10.0` (final) published.
